py-polars/polars/convert.py

```diff
@@ -120,12 +130,15 @@ def from_dicts(
     └─────┴─────┴──────┘
 
     """
-    return DataFrame._from_dicts(dicts, infer_schema_length, schema)
+    return DataFrame._from_dicts(
+        dicts, infer_schema_length, schema_overrides=schema_overrides
+    )
 
 
 def from_records(
     data: Sequence[Sequence[Any]],
     columns: Sequence[str] | None = None,
+    schema_overrides: Schema | None = None,
```

Comment:
Technically, adding this as the third parameter (which makes sense) is a breaking change, in case people were specifying `orient` as a positional parameter.

Not sure if we should worry about that, but if we want to do it nicely, we should add it as the last parameter and then move it to the correct position in the next breaking release.

This same logic affects a lot of the changes in this PR.

Reply:
Ok, I've shifted them to the end; we can pair them back up for 0.16 - next PR will be a straight `columns => schema` update in conjunction with `deprecated_alias`, and then a follow-up can move the params together (with a breaking notification on the commit).

Reply:
I see that the `DataFrame` class constructor still has `schema_overrides` before `orient`. That should really be adjusted, as there must be people initiating their DataFrames with something like `pl.DataFrame([[1,2],[3,4]], ['a', 'b'], 'row')`, which this will break.

Reply:
Shall we make it keyword only?

Reply:
> I see that the `DataFrame` class constructor still has `schema_overrides` before `orient`. That should really be adjusted, as there must be people initiating their DataFrames with something like `pl.DataFrame([[1,2],[3,4]], ['a', 'b'], 'row')`, which this will break.

Doh! Missed it... will fix ASAP; `kwargs`-only doesn't sound like a bad idea?

Reply:
Exactly, we should probably add a few "kwarg only" stars in the next breaking release. You could already make this one kwarg-only.

Reply:
Ok, have updated accordingly - along with making the `infer_schema_lengths` param kwargs-only too. That param did technically sneak out 0.15.15, but it feels more like a kwargs-only param and nobody will be using it yet, so...